src/osd/PrimaryLogPG.h

```diff
@@ -1318,9 +1318,9 @@ class PrimaryLogPG : public PG, public PGBackend::Listener {
   /**
    * scan a (hash) range of objects in the current pg
    *
-   * @begin first item should be >= this value
    * @min return at least this many items, unless we are done
    * @max return no more than this many items
+   * @bi.begin first item should be >= this value
```

Comment:
please note, i don't think this syntax would be supported by doxygen. instead, we should use http://www.doxygen.nl/manual/commands.html#cmdvar or http://www.doxygen.nl/manual/docblocks.html#memberdoc